### PR TITLE
Show placeholder images for game logos

### LINKED ISSUE #11: game_logo() doesn't fail well for games that don't have images
Loads an ugly broken image instead of something nice, or maybe just the title in text.

To reproduce: You'll need to find someone who has a game in common with you that doesn't have a public image. I noticed this on Left 4 Dead 2 Beta, which me and Dave both have.

### PR DESCRIPTION
**Pull request by carsonb**
Also, set the name as the alt text.

Fixes #11 and #12

### CODE REVIEW

File: app/helpers/welcome_helper.rb [hunk 1]
```diff
@@ -1,13 +1,17 @@
 module WelcomeHelper
-  def game_icon(game)
-    image_tag("http://media.steampowered.com/steamcommunity/public/images/apps/#{game.appid}/#{game.img_icon_url}.jpg")
+  def game_icon(game)    
+    if game.img_icon_url.present?
+      image_tag("http://media.steampowered.com/steamcommunity/public/images/apps/#{game.appid}/#{game.img_icon_url}.jpg", alt: game.name)
```
Comment (csaunders) [inline app/helpers/welcome_helper.rb]: I'd move these into the models so you can simply say

`obj.image_url` and it'll do all the if/elses in there.

Less pooplication == win

Comment (carsonb) [inline app/helpers/welcome_helper.rb]: So just add methods to game object and ditch the welcome_helper?

Comment (csaunders) [inline app/helpers/welcome_helper.rb]: Hmmmm.... Maybe we should make those objects consistent then we can just have one function to fetch the image or the placeholder?
- Chris Saunders / sndrs.ca

**Comment by csaunders** [comment]
awesome

**Pull request merged**